contrib/helm/harbor/README.md

```diff
@@ -4,24 +4,75 @@
 
 ## Introduction
 
-This is an experimental monolithic chart that installs and configures VMWare Harbor and its dependencies. The initial implementation of this includes all of the components required to run Harbor. As upstream harbor becomes more cloud native we will be able to break apart the monolith and utitlize helm dependencies.
+This chart installs and configures Harbor.
 
 ## Prerequisites
 
-- Kubernetes 1.7+ with Beta APIs enabled
+- Kubernetes cluster 1.8+ with Beta APIs enabled
+- Kubernetes Ingress Controller is enabled
+- kubectl CLI 1.8+
 - PV provisioner support in the underlying infrastructure
 
+## Setup a Kubernetes cluster
+
+You can use any tools to setup a K8s cluster.
+In this guide, we use [minikube](https://github.com/kubernetes/minikube) to setup a K8s cluster as the dev/test env.
+
+```bash
+# Start minikube
+minikube start --vm-driver=none
```

Comment:
I don't think we should recommend people use `--vm-driver=none` for minikube.

Reply:
Using '--vm-driver=none' to keep it simple to set up.

Reply:
I disagree that this is simple, we should expect anyone using minikube to use it with its default behavior, if people want to use `--vm-driver=none` they're probably an expert user that will know how to deal with the changes.

Reply:
I'm not an expert of minikube, when reading the minikube doc (https://github.com/kubernetes/minikube#linux-continuous-integration-without-vm-support), it tells me to run 'minikube start --vm-driver=none' on Linux, which is very easy and straightforward for me to start a k8s cluster.